pkgs/games/fltrator/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, fetchurl, unzip, fltk, which, libjpeg }:
+
+stdenv.mkDerivation rec {
+  name = "fltrator-${version}";
+  version = "2.3";
+  src = fetchurl {
+    url = "https://sourceforge.net/projects/fltrator/files/v${version}/fltrator-${version}-code.zip/download";
+    sha256 = "125aqq1sfrm0c9cm6gyylwdmc8xrb0rjf563xvw7q28sdbl6ayp7";
+  };
+
+
+  buildInputs = [ fltk unzip which libjpeg ];
+
+  unpackPhase = ''
+    unzip $src
+    '';
```

Comment:
Indent these ending quotes at the same level as installPhase (ie. delete two spaces)